src/commands/synthetics/README.md

```diff
@@ -202,7 +202,8 @@ export DATADOG_SYNTHETICS_LOCATIONS="aws:us-east-1;aws:us-east-2"
 
 ### API
 
-By default, `datadog-ci` runs at the root of the working directory and finds `{,!(node_modules)/**/}*.synthetics.json` files (every file ending with `.synthetics.json`, except for those in the `node_modules` folder). The tool loads `datadog-ci.json`, which can be overridden through the `--config` argument.
+By default, `datadog-ci` runs at the root of the working directory and looks for `{,!(node_modules)/**/}*.synthetics.json` files (every file ending with `.synthetics.json`, except for those in the `node_modules` folder) to find a Test Config file.
+The default value for the Global Configuration file is `datadog-ci.json`. If you name you global config file that you don't need to use the `--config` flag, the file will be automatically picked up.
```

Comment:
```suggestion
The default value for the Global Configuration file is `datadog-ci.json`. If you use this name for your global config file, you don't need to use the `--config` flag, as the file is automatically picked up.
```

Reply:
here would be my suggestion if we are really going to keep that sentence - though I think we are just explaining the concept of a default value 😅 
```suggestion
The default file name for the [global configuration file](#global-configuration-file-options) is `datadog-ci.json`. If you use this name for your global configuration file, you may omit the `--config` flag.
```

Reply:
@Drarig29 thanks, I agree that your suggestion is better in this case (if the decision is to keep the sentence).